mplabx/small-psk-build/psk-ssl.c

```diff
@@ -0,0 +1,7075 @@
+/* psk-ssl.c
```

Comment:
I would prefer to see these more public... src/psk/ssl.c and src/psk/tls.c and support include.am changes to support using these size reductions.